static/app/views/settings/project/server-side-sampling/rule.tsx

```diff
@@ -117,16 +124,33 @@ export function Rule({
           onFinish={() => {
             // TODO(sampling): activate the rule
           }}
-          // TODO(sampling): disable if sdks are not yet updated
-          disabled={true || !isUniform}
+          disabled={!canActivate || !isUniform}
```

Comment:
Let's put this line back, we still want `true` hardcoded for now so that those guides are not visible just yet. 

Later we'll also need to check if the sdk updates are loading or not (otherwise this would become visible while the sdk updates are loading and therefore the array is empty and then disappear)